frontend/packages/ceph-storage-plugin/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,37 @@
+import { execSync } from 'child_process';
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '../../../console-shared/src/test-views/dashboard-shared.view';
```

Comment:
We should avoid using relative paths between packages, `@console/shared/src/test-views/dashboard-shared.view`